last_gas/domain/schedulers/orm.py

```diff
@@ -0,0 +1,53 @@
+from sqlalchemy import Column, String, Integer, DateTime, ARRAY, ForeignKey
+from sqlalchemy.orm import declarative_base, relationship
+
+Base = declarative_base()
+
+
+class SchedulesTypes(Base):
+    __tablename__ = "schedules_types"
+
+    schedule_type_id = Column("schedule_type_id", Integer, primary_key=True)
+    schedule_type = Column("schedule_type", String)
```

Comment:
should we have constraints for that? I mean, only defined values

Reply:
Wouldn't it be better to scale if we were to add new schedule_types to it?